src/high-level/include/iDynTree/KinDynComputations.h

```diff
@@ -371,13 +486,34 @@ class KinDynComputations {
      */
     iDynTree::Transform getWorldTransform(const iDynTree::FrameIndex frameIndex);
 
+    /**
+     * Return the transform where the frame is the frame
+     * specified by frameIndex, and the reference frame is the world one
+     * (world_T_frame).
+     * @param world_T_frame
+     * @warning the Span and the MatrixView objects should point an already existing memory. Memory allocation and resizing cannot be achieved with this kind of objects.
+     * @return true if all went well, false otherwise.
+     */
+    bool getWorldTransform(const iDynTree::FrameIndex frameIndex,
+                           iDynTree::MatrixView<double> world_T_frame);
+
     /**
      * Version of getWorldTransform where the frame is specified by name.
      *
      * @note For real time code use the method that takes an integer.
      *
      */
-    iDynTree::Transform getWorldTransform(std::string frameName);
+    iDynTree::Transform getWorldTransform(const std::string & frameName);
+
+    /**
+     * Version of getWorldTransform where the frame is specified by name
+     * (MatrixView and Span implementation).
```

Comment:
This method takes a MatrixView, not a Span, so drop that from the comment.
Also, specify that the span should point to a 4x4 matrix in row-major format representing a homogenous transformation

Reply:
>  Also, specify that the span should point to a 4x4 matrix in ~~~row-major format~~~ representing a homogenous transformation

👍  for adding the details, however  `MatrixView` support both `row-major` and `column-major`, so there is no need to specify that!